File: crates/metrics/src/throughput_discovery_rate_integrity_contract_model.rs
//! Metrics P32 throughput contract_model discovery-rate integrity feature.
use super::discovery_rate_integrity_support::{manifest,qualify,DiscoveryRateCard7,DiscoveryRateIntegrityError,DiscoveryRateRequest4};
pub const FEATURE_ID:&str="AFA-metrics-P32-F07";pub const CONTRACT_VERSION:&str="metrics-throughput_discovery_rate_integrity_contract_model/1.0";
pub fn throughput_discovery_rate_integrity_contract_model_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"throughput","contract_model")}
pub fn qualify_throughput_discovery_rate_integrity_contract_model(request:&DiscoveryRateRequest4)->Result<DiscoveryRateCard7,DiscoveryRateIntegrityError>{qualify(request,FEATURE_ID,CONTRACT_VERSION,"throughput","contract_model")}

File: python/prism_sdk/throughput_discovery_rate_integrity_contract_model.py
"""Metrics P32 throughput contract_model discovery-rate integrity feature F07."""
from .discovery_rate_integrity_support import DiscoveryRateRequest4,DiscoveryRateCard7,DiscoveryRateIntegrityError,manifest,qualify
FEATURE_ID="AFA-metrics-P32-F07";CONTRACT_VERSION="metrics-throughput_discovery_rate_integrity_contract_model/1.0"
def throughput_discovery_rate_integrity_contract_model_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="contract_model")
def qualify_throughput_discovery_rate_integrity_contract_model(request:DiscoveryRateRequest4)->DiscoveryRateCard7:return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="contract_model")
